PrincessBrideTrivia/PrincessBrideTrivia/Program.cs

```diff
@@ -24,7 +24,7 @@ public static void Main(string[] args)
 
         public static string GetPercentCorrect(int numberCorrectAnswers, int numberOfQuestions)
         {
-            return (numberCorrectAnswers / numberOfQuestions * 100) + "%";
+            return (numberCorrectAnswers * 100.0 / numberOfQuestions) + "%";
```

Comment:
Only thing I would have added here would have been Math.Round before adding the %.

Also, you could have just converted one of these to a float (which is what is happening under the covers when you multiplied by 100.0)

```suggestion
            return Math.Round(numberCorrectAnswers * 100.0 / numberOfQuestions) + "%";
```